docs/config.md

```diff
@@ -26,6 +26,7 @@ This set of variables must be at the top of your TOML file due to not belonging
 - `no_prometheus` (bool) — Toggle prometheus metrics. Default: false.
 - `mount_timeout_sec` (int) — Timeout for mount if a layer can't be resolved. Default: 30.
 - `fuse_metrics_emit_wait_duration_sec` (int) — The wait time before the snaphotter emits FUSE operation counts for an image. Default: 60.
+- `skip_check_snapshotter_supported` (bool) - skip check for snapshotter is supported which can give performance benefits for soci daemon startup time. This config to be done if you are sure overlayfs is supported. Default: false
```

Comment:
nit: soci -> SOCI

nit: "This config to be done if you are sure overlayfs is supported." -> "This config should only be done if you are sure overlayfs is supported."